ports/esp32/modnetwork.c

```diff
@@ -558,6 +569,11 @@ STATIC const mp_map_elem_t mp_module_network_globals_table[] = {
         MP_OBJ_NEW_SMALL_INT(WIFI_AUTH_WPA_WPA2_PSK) },
     { MP_OBJ_NEW_QSTR(MP_QSTR_AUTH_MAX),
         MP_OBJ_NEW_SMALL_INT(WIFI_AUTH_MAX) },
+
+    { MP_OBJ_NEW_QSTR(MP_QSTR_PHY_LAN8720),
+        MP_OBJ_NEW_SMALL_INT(PHY_LAN8720) },
+    { MP_OBJ_NEW_QSTR(MP_QSTR_PHY_TLK110),
+        MP_OBJ_NEW_SMALL_INT(PHY_TLK110) },
```

Comment:
Should the constants be prefixed with something like ETH or LAN to emphasise that they are for wired and not wifi?

Reply:
Well, the hardware in question is a PHY, thus the PHY prefix -- this should stay.

Reply:
But it's specifically a wired LAN PHY, as opposed to a wireless/USB/etc PHY.  So I was thinking something like ETH_PHY_LAN8720.